android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -1058,29 +1058,33 @@ public void onGetUserType(int userType) {
 
     @Override
     public void onBalance(boolean success) {
-        mWalletBalanceLayout.setAlpha(1.0f);
-        mWalletBalanceProgress.setVisibility(View.GONE);
-        if (success) {
-            if (mBraveRewardsNativeWorker != null) {
-                BraveRewardsBalance walletBalanceObject =
-                        mBraveRewardsNativeWorker.GetWalletBalance();
-                double walletBalance = 0;
-                if (walletBalanceObject != null) {
-                    walletBalance = walletBalanceObject.getTotal();
-                }
+        try {
+            mWalletBalanceLayout.setAlpha(1.0f);
```

Comment:
based on the logs, we can just check if mWalletBalanceLayout != null. by catching the whole code block we are avoiding potential misbehaviour. WDYT ?

Reply:
This very rare case where NPE will happen. This case might happened because onBalance is called after onDestroy.
If that is the case even we check one object as null check. Afterwards other object also give NPE because all the case happening after onDestroy.